code/modules/mob/living/carbon/human/species_types/abominations.dm

```diff
@@ -1,18 +1,4 @@
 /// These won't appear normally in games, they are meant to for debuging the adjustment of limbs based on the height of a humans bodyparts.
-/datum/species/human/tallboy
```

Comment:
Should delete all of these species

Reply:
removed the other but left the one that's on birdshot
![image](https://github.com/tgstation/tgstation/assets/53777086/516eb122-9fd0-4d03-8748-f463c79edcb7)

Reply:
IMO just delete the birdshot thing and replace it with a monkey, or if you want to go through the effort, add a mapping helper which replaces limbs of the mob it's placed on

These are just not good species